tests/publiccloud/check_registercloudguest.pm

```diff
@@ -148,6 +147,49 @@ sub check_instance_unregistered {
     }
 }
 
+sub new_registration {
+    my ($instance) = @_;
+    record_info('Starting registration...');
+    $instance->ssh_script_retry(cmd => "sudo ${path}registercloudguest $regcode_param", timeout => 300, retry => 3, delay => 120);
+    check_instance_registered($instance);
+    return 0;
+}
+
+sub test_container_runtimes {
+    my ($instance) = @_;
+    my $image = "registry.suse.com/bci/bci-base:latest";
+
+    record_info('Install docker');
+    $instance->ssh_script_output("sudo rm /root/.docker/config.json");    # workaround for https://bugzilla.suse.com/show_bug.cgi?id=1231185
+    $instance->ssh_script_output("sudo zypper install -y docker");
+    record_info('Test docker');
+    $instance->ssh_script_output("sudo systemctl start docker.service");
+    record_info("systemctl status docker.service", script_output("systemctl status docker.service"));
+    $instance->ssh_script_output("sudo docker pull $image");
+    $instance->ssh_script_output("sudo systemctl stop docker.service");
```

Comment:
Why stop the service?

Reply:
I am just following the written manual instructions.

Reply:
I would agree with Histo here . It is not test which suppose to do something with docker and test instructions which he is trying to implement saying to stop docker . But probably I would same question as above - why script_output ? I would call ssh_assertscript_run here so in case it will fail to stop we might catch this here